gramps/gui/configure.py

```diff
@@ -2069,17 +2058,19 @@ def can_we_use_genealogical_fonts(self, obj):
             font = fontconfig.FcFont(path)
             local = get_env_var('LANGUAGE', 'en')
             if isinstance(font.family, list):
+                # fontconfig version 0.5.x uses a list
                 fontname = None
                 for lang, fam in font.family:
                     if lang == local:
                         fontname = fam
                 if not fontname:
                     fontname = font.family[0][1]
             else:
+                # fontconfig version 0.6.0 uses dict
                 if local in font.family:
                     fontname = font.family[local]
                 else:
-                    for lang, name in font.family:     # version 0.6.0 use dict
+                    for lang, name in font.family:
```

Comment:
I think this needs to be `for lang, name in font.family.items()` otherwise you only get the key, not key and data.

Reply:
done